dist/source/repo-creator/choose-name.ts

```diff
@@ -19,13 +26,18 @@ export class ChooseName {
 		this.templateOwner = parameters.owner;
 		this.templateName = parameters.name;
 	}
-	
+
 	@computedFrom('newRepoName')
 	protected get inputValid() {
 		return !!this.newRepoName;
 	}
 
 	protected createRepository = (): void => {
-		this.router.navigate(`replacements/${this.templateOwner}/${this.templateName}/${this.newRepoName}`);
+		this.validation.validate()
+			.then(() => {
+				this.router.navigate(`replacements/${this.templateOwner}/${this.templateName}/${this.newRepoName}`);
+			})
+			.catch((validationResult: any) => {
+			});
```

Comment:
Please put the the `.then` and `.catch` on the previous lines and shift the indent to the left like this:
```ts
		this.validation.validate().then(() => {
			this.router.navigate(`...`);
		}).catch((validationResult: any) => {
		});
```